tests/unit/applications/test_views.py

```diff
@@ -54,6 +54,47 @@ def test_confirmation_success(rf):
         assert spec.title in response.rendered_content
 
 
+def test_application_records_confirmation_page_reached(rf):
+    user = UserFactory()
+    application = ApplicationFactory(
+        created_by=user,
+    )
+
+    request = rf.get("/")
+    request.user = user
+
+    response = confirmation(request, pk=application.pk)
+    assert response.status_code == 200
+
+    application.refresh_from_db()
+    assert application.has_reached_confirmation
+
+    request = confirmation(request, pk=application.pk)
```

Comment:
did you mean to leave this line in?

Reply:
Good catch; not sure how that one got in there :keyboard::oil_drum:

Reply:
Fixed now :+1:

Reply:
Oh, I remember this: I need to do some kind of check where:
* the confirmation page is loaded and the confirmation page is already reached.
* because otherwise the coverage drops, as there's a code branch not reached.

I'll tidy this up as it was an incomplete thought, but does run all of the code.

Reply:
Actually fixed this time.